pyhik/hikvision.py

```diff
@@ -132,6 +132,77 @@ def current_event_states(self):
         """Return Event states dictionary"""
         return self.event_states
 
```

Comment:
Please add a property here to return the current motion detection state and remain consistent.

```suggestion
@property
def current_motion_detection_state(self):
    """Return current state of motion detection property"""
    return self.motion_detection
```